src/main/java/bio/terra/service/snapshot/flight/delete/DeleteUtils.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.service.snapshot.flight.delete;
+
+import bio.terra.service.snapshot.flight.SnapshotWorkingMapKeys;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+
+public class DeleteUtils {
```

Comment:
Since this is all predicates, it might make sense to name this `SnapshotDeletePredicates` or something along those lines. `Utils` feels a little too general.